nemo/collections/tts/torch/g2ps.py

```diff
@@ -235,3 +237,162 @@ def __call__(self, text):
             prons.extend(pron)
 
         return prons
+
+
+class IPAG2P(BaseG2p):
+    def __init__(
+        self,
+        phoneme_dict,
+        word_tokenize_func=english_word_tokenize,
+        apply_to_oov_word=None,
+        ignore_ambiguous_words=True,
+        heteronyms=None,
+        phoneme_probability: Optional[float] = None,
+    ):
+        """Generic IPA G2P module. This module converts words from grapheme to International Phonetic Alphabet representations.
+        Optionally, it can ignore heteronyms, ambiguous words, or words marked as unchangeable by word_tokenize_func (see code for details).
+        Ignored words are left unchanged or passed through apply_to_oov_word for handling.
+        
+        Args:
+            phoneme_dict (str, Path, Dict): Path to file in CMUdict format or dictionary of CMUdict-like entries.
+                Must be given for IPA G2P. (Consider using scripts/tts_dataset_files/cmudict-0.7b_nv22.06-ipa.)
```

Comment:
do you mean `ipa_cmudict-0.7b_nv22.06.txt`-?

Reply:
Good catch, thanks! Pushing update shortly.